packages/genesys-spark-components/src/components/stable/gux-table-toolbar/gux-table-toolbar.tsx

```diff
@@ -239,6 +240,13 @@ export class GuxTableToolbar {
     });
   }
 
+  // eslint-disable-next-line @typescript-eslint/require-await
+  @Method()
+  async guxForceUpdate(): Promise<void> {
```

Comment:
I initially went down this route but I think I would have to `emit` an event from the `gux-table-toolbar-custom` action and then listen for it in the `gux-table-toolbar` to be able to invoke the layout change methods. I may be wrong though 🤔

Reply:
I think there may also be edge cases where the toolbar may not condense outside of the `slot` change so thats why I went this route.

Reply:
So we will be asking apps to call this method whenever they change something?

Reply:
I think the best way to do this will be to `forceUpdate(this.root)` in the mutation observer method so it will cover all edge cases related to dynamic changes to the DOM tree.